send-client/src/test/java/ai/apptuit/metrics/client/ApptuitPutClientTest.java

```diff
@@ -170,6 +197,53 @@ private void testPut(int status) throws MalformedURLException, ParseException {
     }
   }
 
+  private void testSend(int status, boolean exceptException) throws MalformedURLException, ParseException, ConnectException {
+    testSend(status, exceptException, MockServer.token);
+  }
+
+  private void testSend(int status, boolean exceptException, String token) throws MalformedURLException, ParseException, ConnectException {
```

Comment:
Instead of duplicating, pass a [Function](https://docs.oracle.com/javase/8/docs/api/java/util/function/Function.html) that does either send or put?